Added the book called "Learning Swift: Building Apps for macOS, iOS, …

Sahetdovran opened the pull request:
…and Beyond"

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

Sahetdovran committed:
Added the book called "Learning Swift: Building Apps for macOS, iOS, and Beyond"

amandeepmittal reviewed (changes requested):
This document is poorly formatted. Can you make sure it formats correctly?

Sahetdovran committed:
Tried to format correctly as requested

I tried to format it correctly. I am sorry if I did incorrectly. Could you elaborate on where exactly would you like me to change? Since the first book was there before, I thought it has been accepted, so I tried to format it like that.

Sahetdovran committed:
Merge pull request #1 from Sahetdovran/Sahetdovran-patch-1

Tried to format correctly as requested

Sahetdovran committed:
Tried to format it as it requested

I have tried to format it correctly. i am sorry if I have failed in doing that. If I have failed, could you elaborate on specific place that you want me to change, please? Also, since the first book was already there before, I thought it has been approved so I tried to format the book that I have added.

Sahetdovran commented:
I am not sure if I accomplished what you asked. could you let me know if I had accomplished it, please?

QuincyLarson commented:
Thank you for opening this pull request.

Book recommendations are beyond the scope of the freeCodeCamp Guide, so we've decided to go ahead and remove these book recommendation articles.

I encourage you to review this book in [the ratings category of the freeCodeCamp forum](https://www.freecodecamp.org/forum/c/reviews?order=views). Your recommendation will reach way more people there.

Thanks again recommending this book, and for contributing to the freeCodeCamp Guide.
